rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityList.vue

```diff
@@ -642,6 +663,16 @@ export default defineComponent({
     OffsetPagination,
     ActivityFilter,
   },
+  // props: {
+  //   eventBus: {
+  //     type: Object as PropType<typeof EventBus>,
+  //     required: false,
+  //   },
+  //   displayMode: {
+  //     type: String as PropType<string>,
+  //     required: false,
+  //   },
+  // },
   props: ["eventBus", "displayMode"],
```

Comment:
need to address this part (remove the old structure and uncomment the annotated props)